docker/docker-compose.es.yml

```diff
@@ -2,22 +2,21 @@ version: '3.4'
 
 services:
     es01:
-        image: docker.elastic.co/elasticsearch/elasticsearch:${ES_VERSION:-7.15.2}
-        command: >
+        image: &image docker.elastic.co/elasticsearch/elasticsearch:${ES_VERSION:-7.15.2}
```

Comment:
Yay, YAML magic :-) Have not thought of using this in docker-compose files so far but I see how it removes duplication.